### PR TITLE
Clarify Office Insider requirement

### PR DESCRIPTION
**Pull request by kbrandl**
Addresses issue #477.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/includes/file-tutorial-excel-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/includes/file-tutorial-excel-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810171736077153-479%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OeV5O8gAV4A5%2B%2BTgkYhCXAM1465PNy9G5SbrBlvTMFQ%3D&st=2018-10-17T17%3A32%3A26Z&se=2018-11-17T17%3A37%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-overview.md [hunk 1]
```diff
@@ -373,7 +373,7 @@ function getComment(x) {
 - **May 7, 2018**: Shipped* support for Mac, Excel Online, and synchronous functions running in-process
 - **September 20, 2018**: Shipped support for custom functions JavaScript runtime. For more information, see [Runtime for Excel custom functions](custom-functions-runtime.md).
 
-\* to the Office Insiders Channel
+\* to the [Office Insider](https://products.office.com/office-insider) channel (formerly called "Insider Fast")
```
Comment (AlexJerabek) [inline docs/excel/custom-functions-overview.md]: The terms channel and level are both used to describe Office Insider - Insider. Are they interchangeable?

Comment (kbrandl) [reply docs/excel/custom-functions-overview.md]: Thanks for the feedback @AlexJerabek and nice attention to detail :) Yes, I'd say those terms are interchangeable.  When you join the Office Insider program, you have a choice of two "levels" -- **Insider** and **Monthly channel**.  Until recently, the first "level" was called the "Insider Fast" _channel_ and that's still the term that's most widely recognized.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/includes/file-tutorial-excel-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/includes/file-tutorial-excel-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810171747037399-479%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Y%2BiLMkFe9Wjo%2BfeXRAWF%2FGTmEHO8ZdJ1FqBOfx7TuzY%3D&st=2018-10-17T17%3A43%3A22Z&se=2018-11-17T17%3A48%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/includes/file-tutorial-excel-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-clarify-office-insider-req/docs/includes/file-tutorial-excel-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810171747323926-479%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=s92w6iXm4Zpgd35uE3zaXJq5hVz61TB2FufRLY1cS68%3D&st=2018-10-17T17%3A44%3A37Z&se=2018-11-17T17%3A49%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**